extensions/azurecore/src/account-provider/providerSettings.ts

```diff
@@ -220,7 +220,12 @@ const germanyAzureSettings: ProviderSettings = {
 				endpoint: 'https://vault.microsoftazure.de',
 				azureResourceId: AzureResource.AzureKeyVault
 			},
-			redirectUri: 'https://vscode-redirect.azurewebsites.net/'
+			redirectUri: 'https://vscode-redirect.azurewebsites.net/',
+			scopes: [
+				'openid', 'email', 'profile', 'offline_access',
+				'https://management.microsoftazure.cn/user_impersonation'
```

Comment:
This appears to be using the china endpoint which doesn't seem correct

Reply:
Great catch. Copy paste got me!